packages/x-components/src/design-system-deprecated/components/picture/default.tokens.scss

```diff
@@ -8,16 +8,18 @@
 
   // color
   --x-color-background-picture-default: transparent;
-  --x-color-fill-picture-placeholder-path-default: var(--x-color-base-neutral-70);
-  --x-color-fill-picture-placeholder-rect-default: var(--x-color-base-neutral-95);
-  --x-color-fill-picture-fallback-path-default: var(--x-color-base-neutral-70);
-  --x-color-fill-picture-fallback-rect-default: var(--x-color-base-neutral-95);
 
   // others
   --x-object-fit-picture-default: contain;
   --x-object-fit-picture-fallback-default: var(--x-object-fit-picture-default);
   --x-object-fit-picture-placeholder-default: var(--x-object-fit-picture-default);
-  --x-mix-blend-mode-picture-default: none;
+  --x-mix-blend-mode-picture-default: normal;
   --x-mix-blend-mode-picture-fallback-default: var(--x-mix-blend-mode-picture-default);
   --x-mix-blend-mode-picture-placeholder-default: var(--x-mix-blend-mode-picture-default);
+
+  // color
+  --x-color-fill-picture-placeholder-path-default: var(--x-color-base-neutral-70);
+  --x-color-fill-picture-placeholder-rect-default: var(--x-color-base-neutral-95);
+  --x-color-fill-picture-fallback-path-default: var(--x-color-base-neutral-70);
+  --x-color-fill-picture-fallback-rect-default: var(--x-color-base-neutral-95);
```

Comment:
F! When the doubts I left in comments are solved, I will move this part "upstairs" at line 11 with the other color token